cmd/report_test.go

```diff
@@ -46,4 +49,5 @@ func TestCreateReport(t *testing.T) {
 	assert.EqualValues(t, 6, m["vus_max"])
 	assert.EqualValues(t, 170, m["iterations"])
 	assert.NotEqual(t, "0s", m["duration"])
+	assert.EqualValues(t, map[string]any{"ci": true}, m["env"])
```

Comment:
I'd rather suggest testing that by default the CI's value is `false` and introduce a dedicated test case, where we can test that in the CI environment we do reporting that it sets.